### DISCUSSION TITLE
function upon q-option-group selection

### DISCUSSION THREAD

**Discussion by smithGit** [quasarframework/quasar#11563] [General - Components / Directives / etc] [answered]
<q-card-section>
          <q-option-group :options="chapters" type="radio" v-model="selChapter" color="primary">
              @update:model-value=function(){ alert("I got it!") }
          </q-option-group>
          <div>SEL CHPT IS: {{ selChapter }}</div>
        </q-card-section>
In the code above, selChapter is being set upon selection, but I cannot find how to trigger an event function.  @update:model-value nor @input work.  Help would be appreciated!

**Comment by pdanpdan** [comment, marked as answer]
You wrote the handler as the content of the tag, not inside the tag

**Reply by smithGit** [reply]
Thanks for your propmt response to this (unnecessary) query.  Terribly sorry for this oversight.  If possible please delete this query.  All works as documented!
